### ISSUE TITLE
ORT Scanner: invalid spec

### ISSUE THREAD

**Issue by elgohr** [oss-review-toolkit/ort-ci-github-action#37]
Adding scanner into the run parameter results in 
```
 Running ORT Scanner... 
docker: invalid spec: :/ssh.socket: empty section between colons.
See 'docker run --help'.
Error: Process completed with exit code 125.
```
Used configuration as
```yaml
name: Ort
on: 
  push:
    branches: [ main ]
  pull_request:
    branches: [ main ]
permissions:
  contents: read
jobs:
  test:
    runs-on: ubuntu-latest
    timeout-minutes: 5
    steps:
      - uses: actions/checkout@v4
      - run: git config --global url.https://github.com/.insteadOf ssh://git@github.com/
      - uses: oss-review-toolkit/ort-ci-github-action@v1
        with:
          run: 'cache-dependencies,analyzer,scanner,evaluator,advisor,reporter,upload-results'
      
```

**Comment by elgohr** [comment]
I guess it's because of the usage of `SSH_AUTH_SOCK` that hadn't been set

**Comment by cd-dev-ge** [comment]
Same here. How do we fix it?

**Comment by sschuberth** [comment]
> I guess it's because of the usage of `SSH_AUTH_SOCK` that hadn't been set

Right, which in turn means that `ssh-agent` was not started. Please give https://github.com/oss-review-toolkit/ort-ci-github-action/pull/38 a try.

**Comment by cd-dev-ge** [comment]
It works. Thanks.

**Comment by marcherdina** [comment]
Unfortunately it doesn't work for me:

```
 Running ORT Scanner... 
docker: invalid spec: :/ssh.socket: empty section between colons.
See 'docker run --help'.
Error: Process completed with exit code 125.
```

That's my Action file:

```
name: OSS-ORT GitAction
 
on:
  workflow_dispatch:
jobs:
  ort:
    runs-on: ubuntu-latest
    name: Run ORT

    steps:
      - name: Use HTTPS with personal token always for Git cloning
        run: |
          git config --global url."https://oauth2:${{ secrets.ORT_CONFIG_REPO_TOKEN }}@github.com/".insteadOf "ssh://git@github.com/"
      - name: Checkout project
        uses: actions/checkout@v3
      - name: Run GitHub action for ORT
        uses: oss-review-toolkit/ort-ci-github-action@v1
        with:
          ort-config-repository: 'https://oauth2:${{ secrets.ORT_CONFIG_REPO_TOKEN }}@github.com/marcherdina/ort-github.git'
          run: >
            cache-dependencies,
            metadata-labels,
            analyzer,
            scanner,
            evaluator,
            advisor,
            reporter,
            upload-results
          report-formats: >
            SpdxDocument,WebApp
```

**Comment by sschuberth** [comment]
@marcherdina, please double-check that `eval "$(ssh-agent -s)"` sets the `SSH_AUTH_SOCK` environment variable in your case. (Also inspect the output of just `ssh-agent -s` for debugging.)

**Comment by mschuettlerTNG** [comment]
Hi @marcherdina ,
I just ran into the same issue. Turns out, the fix from #38 is not on `@v1`. I switched to `uses: oss-review-toolkit/ort-ci-github-action@main` for now

**Comment by sschuberth** [comment]
> Turns out, the fix from [#38](https://github.com/oss-review-toolkit/ort-ci-github-action/pull/38) is not on `@v1`.

FYI, I've just pushed a `v1.1.0` tag (which now `v1` points to), as I believe [enough fixes have accumulated](https://github.com/oss-review-toolkit/ort-ci-github-action/compare/v1.0.1...v1.1.0).
